openverse_catalog/dags/provider_api_scripts/test_jamendo.py

```diff
@@ -1,15 +1,13 @@
 import json
 import logging
-import os
+from pathlib import Path
 from unittest.mock import patch
 
 import jamendo
 from common.licenses.licenses import LicenseInfo
 
 
-RESOURCES = os.path.join(
-    os.path.abspath(os.path.dirname(__file__)), "tests/resources/jamendo"
-)
+RESOURCES = Path(__file__).parent.resolve() / "tests/resources/jamendo"
```

Comment:
I didn't know that the `/` operator worked on paths. This is very cool!

Reply:
Another cool part is that `pathlib` automatically converts the separators for other system (eg '/' -> '\' for Windows) !